users/v1/scripts/common.py

```diff
@@ -0,0 +1,24 @@
+from functools import partial
+
+def lines(*args: str) -> str:
+    """Create a multi-line string based on input lines."""
+    return "\n".join([*args])
+
+__HEADER_WARNING = """/*
+**  /!\\ This file is generated with the script `{script_path}/{main_file}`
+**  Do NOT manually modify or create files in the `{generated_folder}` folder.
+**  Prefer re-running the script: `python {main_file}`
+**  ex:
+**
+**       $> python {main_file}
+**
+**  Copyright {year} Pierre-Nicolas "Ark'Anoryn" Sormani <arkanoryn@gmail.com>
+**  SPDX-License-Identifier: GPL-2.0-or-later
+**
+*/
+"""
+
+_HEADER = partial(__HEADER_WARNING.format, year="2024", main_file="main.py", generated_folder="generated/", script_path="scripts")
+
+HEADER_CFILE = _HEADER()
```

Comment:
The point of using `partial` is to "capture" some arguments for the function (common ones), and later do actual executions of the function with extra (callsite-specific) ones. Since you are doing a single call you should just 
```suggestion
HEADER_CFILE = __HEADER_WARNING.format(year="2024", main_file="main.py", generated_folder="generated/", script_path="scripts")
```